[ADD] rental: Added pickup and return page (#19590)
odoo/documentation: content/applications/sales/rental/rental_orders/pickup_return.rst

materials and roles <rental/service_products/materials-and-roles>` in the **Planning** app before
creating the product.

Non-physical service products also require the **Timesheets** app to track services that use

> time-based pricing rates accurately. For non-physical service products, :ref:`configure employee

Reviewer: ```suggestion
Non-physical service products also require the **Timesheets** app to accurately track services that use
time-based pricing rates. For non-physical service products, :ref:`configure employee
```

roles <rental/labor_service_products/configure-employee-roles>` in the **Planning** app before
creating the product.

.. _rental/pickup_return/ecommerce-products:

Configure pick up and return availability for eCommerce products
================================================================

> The **Rental** app supports eCommerce configuration at :ref:`a company

Reviewer: ```suggestion
The **Rental** app supports eCommerce configuration at :ref:`the company
```

<rental/product_type/configuration>` and product level when the **Website** app and the

> **eCommerce** module are installed. Using the *Availabilities* setting restricts the days and time

Reviewer 2: ```suggestion
**eCommerce** module are installed. Using the *Availabilities* setting restricts the days and times
```

> the company allows pickup and returns to be scheduled through the online store. To configure the

Reviewer 2: ```suggestion
the company allows pickups and returns to be scheduled through the online store. To configure the
```

   meeting rooms, and storage spaces, the pickup (check-in) and return (check-out) processes are the
   same.

> To process a pickup of a rental product, go to :menuselection:`Rental app --> Orders` and select the

Reviewer: ```suggestion
To process a pickup of a rental product, go to :menuselection:`Rental app --> Orders --> Orders` and select the
```
Please remember to check the menu paths. This *was* correct in the previous version, and it was removed with this edit.

desired rental order. Click :guilabel:`Pickup`, and a *Validate a pickup* pop-up window displays,
listing the reserved rental product and quantity in :guilabel:`Product`, :guilabel:`Reserved`, and
:guilabel:`Picked-up` columns.

.. image:: pickup_return/validate-a-pickup-window.png
   :alt: Example of a Validate a pickup page in the Rental app.

To confirm the list, click :guilabel:`Validate`. The action updates the rental order with a
:guilabel:`Picked-up` badge and updates the :guilabel:`Delivered` column. The action is also

> recorded in the rental order’s Chat.

Reviewer: ```suggestion
recorded in the rental order's chatter.
```

   form.

To process a pickup of a physical rental product with tracking enabled, go to :menuselection:`Rental

> app --> Orders` and select the desired rental order. Click :guilabel:`Pickup`, and the *Validate a

Reviewer: ```suggestion
app --> Orders --> Orders` and select the desired rental order. Click :guilabel:`Pickup`, and the *Validate a
```

pickup* pop-up window displays, listing the reserved rental product and quantity in
:guilabel:`Product`, :guilabel:`Reserved`, :guilabel:`Serial Numbers`, and :guilabel:`Picked-up`
columns. Check the rental product’s serial number and then click in the :guilabel:`Serial Numbers`

> column and select the product’s serial number from the drop-down list.

Reviewer: ```suggestion
columns. Check the rental product's serial number and then click in the :guilabel:`Serial Numbers`
column and select the product's serial number from the drop-down list.
```

.. image:: pickup_return/validate-pickup-with-serial-numbers.png
   :alt: Example of a Validate a pickup page with product tracking enabled in the Rental app.

To confirm the list, click :guilabel:`Validate`. The action updates the rental order with a
:guilabel:`Picked-up` badge and updates the :guilabel:`Delivered` column. The action is also

> recorded in the rental order’s Chat.

Reviewer: ```suggestion
recorded in the rental order's chatter.
```

entering time on the associated task (if applicable).

If time is entered on the *Timesheets* tab of an associated task before the physical rental product

> is picked up, the rental order automatically displays a :guilabel:`Picked-up` badge, however the

Reviewer: ```suggestion
is picked up, the rental order automatically displays a :guilabel:`Picked-up` badge; however, the
```

   both must be returned on the rental order. Returning a rental service is the same process as
   returning a physical rental product without tracking enabled.

> When a customer returns the rental product, go to :menuselection:`Rental app --> Orders` and select

Reviewer: ```suggestion
When a customer returns the rental product, go to :menuselection:`Rental app --> Orders --> Orders` and select
```

the desired rental order. Click :guilabel:`Return`, and the *Validate a return* pop-up window
displays, listing the returning rental product and quantity in :guilabel:`Product`,
:guilabel:`Picked-up`, and :guilabel:`Returned` columns.

.. image:: pickup_return/validate-a-return-window.png
   :alt: Example of a Validate a return page in the Rental app.

To confirm the list, click :guilabel:`Validate`. The action applies a :guilabel:`Returned` badge to
the rental order and updates the :guilabel:`Returned` column. The action is also recorded in the

> rental order’s Chat.

Reviewer: ```suggestion
rental order's chatter.
```

Return a physical rental product with tracking enabled
------------------------------------------------------

> When a customer returns the product, go to :menuselection:`Rental app --> Orders` and select the

Reviewer: ```suggestion
When a customer returns the product, go to :menuselection:`Rental app --> Orders --> Orders` and select the
```

desired rental order. Click :guilabel:`Return`, and the *Validate a return* pop-up window displays,
listing the returning rental product and quantity in :guilabel:`Product`, :guilabel:`Picked-up`,
:guilabel:`Serial Numbers`, and :guilabel:`Returned` columns.

.. image:: pickup_return/validate-a-return-window-with-serial-numbers.png
   :alt: Example of a Validate a return page with product tracking enabled in the Rental app.

Verify that the returned rental product has the same serial number as the one listed in the

> :guilabel:`Serial Numbers` column. If the serial numbers don’t match, click into the

Reviewer: ```suggestion
:guilabel:`Serial Numbers` column. If the serial numbers don't match, click into the
```

:guilabel:`Serial Numbers` column and select the correct serial number from the drop-down menu.

Once the rental product's quantity and serial number match, click :guilabel:`Validate`. The action
applies a :guilabel:`Returned` badge to the rental order and updates the :guilabel:`Returned`

> column. The action is also recorded in the rental order’s Chat.

Reviewer: ```suggestion
column. The action is also recorded in the rental order's chatter.
```
